traits/Cargo.toml

```diff
@@ -11,7 +11,10 @@ repository = "https://github.com/franziskuskiefer/hpke-rs"
 
 [dependencies]
 serde_json = { version = "1.0" }
```

Comment:
Do we still need `serde_json`? I think this could go.

Reply:
I didn't remove it in this PR because #47 removed it and I wanted to avoid a merge conflict (which still happened)